slugid.js

```diff
@@ -42,13 +52,15 @@ exports.decode = function(slug) {
   return uuid.unparse(new Buffer(base64, 'base64'));
 };
 
-/** Generate a v4 (random) uuid and encode it to a slug */
+/** Returns a randomly generated uuid v4 compliant slug guaranteed to begin
+ * with [A-Za-f] */
 exports.v4 = function() {
   var bytes   = uuid.v4(null, new Buffer(16));
+  bytes[0] = bytes[0] & 0x7f;  // unset first bit to ensure [A-Za-f] first char
```

Comment:
For doing something ugly this is the right way to do it :)

I didn't know this would actually work... base64 seems "magic" to me.